src/scenes/AcceleratorRouter/People/PersonForm.tsx

```diff
@@ -6,31 +6,47 @@ import { MultiSelect, Textfield } from '@terraware/web-components';
 import Card from 'src/components/common/Card';
 import PageForm from 'src/components/common/PageForm';
 import { useLocalization, useUser } from 'src/providers/hooks';
+import { PersonData } from 'src/scenes/AcceleratorRouter/People/PersonContext';
 import strings from 'src/strings';
+import { DeliverableCategories, DeliverableCategoryType, categoryLabel } from 'src/types/Deliverables';
 import { USER_GLOBAL_ROLES, UserWithGlobalRoles, getGlobalRole } from 'src/types/GlobalRoles';
 import { User, UserGlobalRole } from 'src/types/User';
 import { isAllowed } from 'src/utils/acl';
 import useDeviceInfo from 'src/utils/useDeviceInfo';
 
 type PersonFormProps = {
-  busy?: boolean;
   emailEnabled?: boolean;
   emailError?: string;
-  user?: User;
+  personData?: PersonData;
   onCancel: () => void;
-  onChange?: (person: User) => void;
-  onSave: (person: User) => void;
+  onChange?: (person: PersonData) => void;
```

Comment:
I think this part is a bit confusing. The "record" used in a form generally shouldn't contain functions in it, it should be a somewhat standard key value pair. I see that you are trying to pass around both the `user` and the `deliverableCategories` as separate objects for convenience, but in my opinion it makes the code a bit more tightly coupled. 

I think the proper way to go about this is to have a `UserWithDeliverableCategories` type that the form is concerned with, and let the provider or hook that is responsible for updating pull the data apart and make the separate API calls. This way the form can be unaware of how the data is ultimately stored.